go/src/koding/klientctl/machine.go

```diff
@@ -255,22 +256,39 @@ func tabListMountFormatter(w io.Writer, mounts map[string][]sync.Info) {
 	fmt.Fprintf(tw, "ID\tMACHINE\tMOUNT\tFILES\tQUEUED\tSYNCING\tSIZE\n")
 	for alias, infos := range mounts {
 		for _, info := range infos {
-			fmt.Fprintf(tw, "%s\t%s\t%s\t%d/%d\t%d\t%d\t%s/%s\n",
+			sign := info.Syncing
+			fmt.Fprintf(tw, "%s\t%s\t%s\t%s/%s\t%s\t%s\t%s/%s\n",
 				info.ID,
 				alias,
 				info.Mount,
-				info.Count,
-				info.CountAll,
-				info.Queued,
-				info.Syncing,
-				humanize.IBytes(uint64(info.DiskSize)),
-				humanize.IBytes(uint64(info.DiskSizeAll)),
+				dashIfNegative(sign, info.Count),
+				dashIfNegative(sign, info.CountAll),
+				dashIfNegative(sign, info.Queued),
+				errorIfNegative(info.Syncing),
+				dashIfNegative(sign, humanize.IBytes(uint64(info.DiskSize))),
+				dashIfNegative(sign, humanize.IBytes(uint64(info.DiskSizeAll))),
 			)
 		}
 	}
 	tw.Flush()
 }
 
+func errorIfNegative(val int) string {
+	if val < 0 {
+		return "err"
```

Comment:
It won't, probably we need to start syncer even if there is an error and work in "offline" mode. But this needs https://github.com/koding/koding/issues/10750 to be implemented first.